scico/test/test_numpy_util.py

```diff
@@ -192,3 +193,10 @@ def test_broadcast_nested_shapes():
         (1, 2, 3),
         (1, 7, 4, 3),
     )
+
+
+def test_is_scalar_equiv():
+    assert is_scalar_equiv(1e0)
+    assert is_scalar_equiv(snp.array(1e0))
```

Comment:
throw in `assert is_scalar_equiv(snp.sum(snp.zeros(1)))` just to directly address the original cause of all this?

Reply:
Good idea. Done.